app/models/remote_sandbox.rb

```diff
@@ -76,10 +84,21 @@ def send_submission(submission, notify_url)
         submission.save!
       rescue StandardError => e
         puts e
-        raise SandboxUnavailableError
+        begin
+          # Try to parse the response to see if the error originated from within the sandbox.
+          @parsed = JSON.parse e.response
+        rescue StandardError
+          raise SandboxUnavailableError
```

Comment:
Make sure submission is tried again if the sandbox is full: https://github.com/rage/sandbox/blob/2e1c4d754cde345acd9bf9b3fd51ae578ea5195c/src/middleware/error_handler.ts#L35